### ISSUE TITLE
Any plans to explore using sinc filter for downsampling?

### ISSUE THREAD

**Issue by PeterL1n** [adobe/antialiased-cnns#46]
Awesome work.

Take a look at alias-free gan by Nvidia if you haven't already.
https://nvlabs.github.io/alias-free-gan/

The filters in this research only reduce alias by smoothing but using a sinc kaiser filter like alias-free gan can almost completely remove aliasing. It will be very interesting to see how the network performs.

**Comment by richzhang** [comment]
Yes, that's a great paper! I think better filters should indeed help
